chef-provisioning.gemspec

```diff
@@ -19,6 +19,7 @@ Gem::Specification.new do |s|
   s.add_dependency 'inifile', '~> 2.0'
   s.add_dependency 'cheffish', '~> 1.1'
   s.add_dependency 'winrm', '~> 1.3'
+  s.add_dependency 'mixlib-install'
```

Comment:
Not sure there's a useful version constraint to be had right now - I can lock it to `<= 1.0` but that doesn't actually provide any useful dependency data.